### PR TITLE
Sprint 3 issue 13 createwhereareyounowscreen

### PR DESCRIPTION
**Pull request by landillo**
#What did we do?

We created the "Where are you now?" screen and implemented. the necessary functionality, where we can choose a flight origin, view the selection result, and navigate to the next screen.

We also created the screen to select the flight destination and its functionality, where we can choose a destination and view the result.
Screenshot_1688778519
Screenshot_1688778549
Screenshot_1688778558
Screenshot_1688778563
Screenshot_1688778573

### CODE REVIEW

**Review by AlbertoG22 — APPROVED** [review]
Excellent job team, keep it up! 😀

File: ios/Podfile.lock [hunk 1]
```diff
@@ -425,6 +425,8 @@ PODS:
     - React-Core
   - RNCMaskedView (0.2.9):
     - React-Core
+  - RNCPicker (2.4.10):
+    - React-Core
```
Comment (ecruz17) [inline ios/Podfile.lock:429]: I'd suggest to gitignore this file ;)

File: package-lock.json
Comment (ecruz17) [inline package-lock.json:1]: and gitignore this file too ^^

File: src/navigation/StackNavigator.tsx [hunk 2]
```diff
@@ -1,27 +1,32 @@
 import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
-import SignUpScreen from '../screens/SignUpScreen';
-import { LogInScreen } from '../screens/LogInScreen';
+//import SignUpScreen from '../screens/SignUpScreen';
+//import { LogInScreen } from '../screens/LogInScreen';
 import { Dimensions } from 'react-native';
```
Comment (ecruz17) [inline src/navigation/StackNavigator.tsx]: You should delete your unused imports

**Review by ecruz17 — APPROVED** [review]
Nice work! 🚀

File: src/appTheme/AppTheme.tsx
Comment (FranciscoSenestrari) [inline src/appTheme/AppTheme.tsx:1]: Delet comment

File: src/components/FlightInfo.tsx
Comment (FranciscoSenestrari) [inline src/components/FlightInfo.tsx:1]: Delet Comments and unused code

File: src/components/OriginDropDown.tsx
Comment (FranciscoSenestrari) [inline src/components/OriginDropDown.tsx:1]: Formta code

File: src/interfaces/interfaces.tsx
Comment (FranciscoSenestrari) [inline src/interfaces/interfaces.tsx:1]: delet unused code

**Review by FranciscoSenestrari — CHANGES_REQUESTED** [review]

**Review by ShadowEnchanter — APPROVED** [review]
Good Job!

File: src/navigation/StackNavigator.tsx
Comment (FranciscoSenestrari) [inline src/navigation/StackNavigator.tsx:1]: needs delete comments in the code

File: src/interfaces/interfaces.tsx
Comment (FranciscoSenestrari) [inline src/interfaces/interfaces.tsx:1]: delete comments

File: src/appTheme/AppTheme.tsx
Comment (FranciscoSenestrari) [inline src/appTheme/AppTheme.tsx:1]: delete comments

**Review by FranciscoSenestrari — APPROVED** [review]

**Pull request merged**